src/System.Private.ServiceModel/tests/Common/Infrastructure/src/ServiceUtilHelper.cs

```diff
@@ -0,0 +1,202 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
```

Comment:
Reviewers: the code here used to be in Endpoints.cs, which wasn't really the right home, and on which this project could not depend.  It has been moved verbatim (but renamed) from Endpoints.cs into this project.  Comments welcome, of course, but all the code in this class already existed.